core/remote/watcher.js

```diff
@@ -416,7 +424,22 @@ class RemoteWatcher {
         break
       case 'DirMove':
         log.info({path, oldpath: change.was.path}, 'folder was moved or renamed remotely')
-        await this.prep.moveFolderAsync(sideName, change.doc, change.was)
+        const needUpdates /*: any */ = []
+        const newRemoteRevs /*: RemoteRevisionsByID */ = {}
+        for (let descendant of (change.descendantDirMoves || [])) {
```

Comment:
Here I believe we're missing the fact that each item of `descendantDirMoves` can actually have `descendantDirMoves` too (cf [proposals above](https://github.com/cozy-labs/cozy-desktop/pull/1250/files#r233131826)).

Or did I miss something?